include/eve/deps/kumi/tuple.hpp

```diff
@@ -707,18 +707,19 @@ namespace kumi
     requires(sizeof...(Ts) == sizeof...(Us) && _::piecewise_ordered<tuple, tuple<Us...>>)
     {
       auto res = get<0>(lhs) < get<0>(rhs);
+
       auto const order = [&]<typename Index>(Index i)
       {
         auto y_less_x_prev  = rhs[i]  < lhs[i];
         auto x_less_y       = lhs[index_t<Index::value+1>{}] < rhs[index_t<Index::value+1>{}];
-        res                 = res || (x_less_y && !y_less_x_prev);
+        return (x_less_y && !y_less_x_prev);
```

Comment:
What's with the kumi changes.

Reply:
these are for doign a proper fold instead of the bad code from before. This is coming from KUMI head